prow/plugins/lgtm/lgtm.go

```diff
@@ -48,7 +48,7 @@ var (
 	// LGTMRe is the regex that matches lgtm comments
 	LGTMRe = regexp.MustCompile(`(?mi)^/lgtm(?: no-issue)?\s*$`)
 	// LGTMCancelRe is the regex that matches lgtm cancel comments
-	LGTMCancelRe        = regexp.MustCompile(`(?mi)^/lgtm cancel\s*$`)
+	LGTMCancelRe        = regexp.MustCompile(`(?mi)^/remove-lgtm|lgtm cancel\s*$`)
```

Comment:
I think we need to parenthesize the actual commands so that the `\s*$` part applies on remove-lgtm too

Reply:
Oh that makes sense, thanks for the feedback! Will make the changes 😄